src/com/lftechnology/java/training/srijan/basics/Multiples.java

```diff
@@ -0,0 +1,50 @@
+package com.lftechnology.java.training.srijan.basics;
+
+/**
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+   Find the sum of all the multiples of 3 or 5 below 1000.
+ */
+
+import java.util.logging.Logger;
+
+public class Multiples {
+	private int num = 1000;
+	private int sum = 0;
+	
+	private static final Logger LOGGER = Logger.getLogger(Fabonacci.class.getName());
```

Comment:
What is Fabonacci.class.getName().Use the same class name you are using the logger for.